docs/source/developer_guide/documentation.rst

```diff
@@ -22,15 +22,20 @@ sktime uses the numpydoc_ Sphinx extension and follows
 `NumPy docstring format <https://numpydoc.readthedocs.io/en/latest/format.html>`.
 
 To ensure docstrings meet expectations, sktime uses a combination of validations built into numpydoc_,
-pydocstyle_ (set to the NumPy convention) pre-commit checks, automated testing of docstring examples to ensure
-the code runs without error, and :ref:`reviewer <reviewer_guide_doc>` feedback.
+pydocstyle_ (set to the NumPy convention) pre-commit checks and automated testing of docstring examples to ensure
+the code runs without error. However, the automated docstring validation in pydocstyle_ only covers basic formatting.
+Passing these tests is necessary to meet the sktime docstring conventions, but is not sufficient for doing so.
+
+To ensure docstrings meet sktime's conventions, developers are expected to check their docstrings against numpydoc_
+and sktime conventions and :ref:`reviewer's <reviewer_guide_doc>` are expected to also focus feedback on docstring
+quality.
 
 sktime Specific Conventions
 ---------------------------
```

Comment:
I think it would be good to highlight some good examples. Something like this:
```suggestion

Examples of Good Docstrings
-----------------------------
To help you understand the docstring specifications, here are a few good examples:

* add link to boss classification
* add link to theta forecaster
* add link to something else which is considered good

sktime Specific Conventions
---------------------------
```

Reply:
I've gone back and forth whether to link specific docstrings here or create an example. 

Upside of linking specific docstrings is that we don't have to generate an example. Downside is that we need to keep an eye on the linked docstrings so we don't end up linking to docstrings that get messed up, etc. 

I'm leaning toward including the direct links for simplicity. What does everyone else think?

Reply:
I would say linking is fine

Reply:
I've added a section on this as suggested.